src/state_control.cpp

```diff
@@ -39,7 +39,7 @@ using namespace std;
 #define RESET "\e[0m"
 
 // State machine
-enum controller_state
+enum tracker_state
```

Comment:
Yeah, I guess my opposition is mainly because I don't think
"controller_state" or "tracker_state" are correct. Just a naming convention
thing.

On Fri, Aug 28, 2015 at 6:26 PM James Svacha <notifications@github.com>
wrote:

> In src/state_control.cpp
> <https://github.com/justinthomas/ground_effect/pull/6#discussion_r38250513>
> :
>
> > @@ -39,7 +39,7 @@ using namespace std;
> >  #define RESET "\e[0m"
> >
> >  // State machine
> > -enum controller_state
> > +enum tracker_state
>
> It doesn't need to. I just did it for consistency. It seems like you are
> calling controllers trackers now.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/justinthomas/ground_effect/pull/6/files#r38250513>.
>